dev-tools/test.sh

```diff
@@ -6,17 +6,28 @@
 # shellcheck source=./dev-tools/_functions.sh
 source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
 
-# Delete outdated code coverage
-rm -rf "${BASE_DIR:?}/htmlcov/"
+# Delete outdated code coverage report
+CODE_COVERAGE_DIR="${BASE_DIR:?}/htmlcov"
+rm -rf "${CODE_COVERAGE_DIR}"
 
 require_installed
 require_database
 
 # Set dummy key to enable SUMM.AI during testing
 export INTEGREAT_CMS_SUMM_AI_API_KEY="dummy"
 
-deescalate_privileges pipenv run pytest --disable-warnings --quiet --numprocesses=auto --cov=integreat_cms --cov-report html
+# If only particular tests should be run, test path can be passed as CLI argument
+TEST_PATH=$1;
+
+if [[ -z "$TEST_PATH" ]]; then
+    deescalate_privileges pipenv run pytest --disable-warnings --quiet --numprocesses=auto --cov=integreat_cms --cov-report html
+else
+    deescalate_privileges pipenv run pytest "$TEST_PATH" --disable-warnings --quiet --numprocesses=auto
```

Comment:
```suggestion
    echo -e "Running all tests..." | print_info
    deescalate_privileges pipenv run pytest --disable-warnings --quiet --numprocesses=auto --cov=integreat_cms --cov-report html
elif [[ -e "$TEST_PATH" ]]; then
    echo -e "Running tests in ${TEST_PATH}..." | print_info
    deescalate_privileges pipenv run pytest "$TEST_PATH" --quiet --disable-warnings --numprocesses=auto
else
    echo -e "${TEST_PATH}: No such file or directory" | print_error
    exit 1
```
Maybe we could show an error if the given path does not exist?
Otherwise the output is just
```
bringing up nodes...

11985 warnings in 5.42s
```
without any error or success message, which could be misinterpreted as "nothing went wrong"...

Reply:
Agree